Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -963,11 +963,11 @@ namespace OpenInfraPlatform {
 					
 					// Calculate an angle on the circle (with circle center in (0., 0., 0.)) for trimming begin.
 					carve::geom::vector<3> point = getPointOnCurve<typename IfcEntityTypesT::IfcCircle>(circle, trim1Vec, trimmingPreference);
-					double startAngle = getAngleOnCircle(carve::geom::VECTOR(0., 0., 0.), circleRadius, point);
+					double startAngle = getAngleOnCircle(conicPositionMatrix * carve::geom::VECTOR(0., 0., 0.), circleRadius, point);
 					
 					// Calculate an angle on the circle (with circle center in (0., 0., 0.)) for trimming end.
 					point = getPointOnCurve<typename IfcEntityTypesT::IfcCircle>(circle, trim2Vec, trimmingPreference);
-					double endAngle = getAngleOnCircle(carve::geom::VECTOR(0., 0., 0.), circleRadius, point);
+					double endAngle = getAngleOnCircle(conicPositionMatrix * carve::geom::VECTOR(0., 0., 0.), circleRadius, point);

```

Comment:
I'd assume that the function `getAngleOnCircle` does not take into account if the circle is rotated ...

Reply:
Resolved:
0f25f00